dev/buildtool/spin_commands.py

```diff
@@ -283,6 +324,12 @@ def __init__(self):
   def init_argparser(self, parser, defaults):
     super(PublishSpinCommandFactory, self).init_argparser(
         parser, defaults)
+    self.add_argument(
```

Comment:
why "build" and not "publish"?

Reply:
It's the same bucket as build. I can rename it to `spin_bucket` or something similar.

Reply:
I didnt see that. It might be ok. I think I've done that in places too. Though I do like spin_bucket better since it is context-free.

Reply:
Changed.